test/expected/normalized.out

```diff
@@ -23,36 +23,33 @@ SELECT create_prometheus_table('input');
 (3 rows)
 
 \d input
-              View "public.input"
- Column |           Type           | Modifiers 
---------+--------------------------+-----------
- sample | prom_sample              | 
- time   | timestamp with time zone | 
- name   | text                     | 
- value  | double precision         | 
- labels | jsonb                    | 
-Triggers:
-    insert_trigger INSTEAD OF INSERT ON input FOR EACH ROW EXECUTE PROCEDURE prometheus.insert_view_normal('input_values', 'input_labels')
+                        View "public.input"
```

Comment:
Beware that this output is different across different versions of PostgreSQL/psql. So, tests might break if run on a different (older) version of PostgreSQL.

I think we should set a minimum supported PostgreSQL version or make this output PG-version independent (like we've done for TimescaleDB with a custom function).